package.json

```diff
@@ -10,8 +10,11 @@
     "build:run": "nodemon app.js",
     "build": "npm run clean; NODE_ENV=production webpack --color --config webpack.prod.js",
     "clean": "rimraf build",
-    "docker:debug": "docker-compose -f docker-compose.debug.yml -p example_webapp_debug up",
-    "docker:prod": "docker-compose -f docker-compose.prod.yml -p example_webapp_prod up"
+    "dev": "npm run build:watch & npm run build:run",
```

Comment:
This is the hacky solution: doing build:watch asyncronously and build:run in the foreground. So webpack watch AND nodemon are both running, each hot reloading the part it's responsible for. Never need to install node modules on the host machine at all!

Well you need to do that for vscode, but I intend to figure out how to develop in the container too so you really don't touch the host.